core/input/generisdata_tech.prn

```diff
@@ -61,7 +61,7 @@ constrTme            4           1           5            1            4
 mix0
 eta                 1.00        1.00        1.00        1.00         1.00       1.00        1.00        1.00
 omf                 0.04        0.06       0.025        0.02         0.03       0.02                   0.025
-lifetime              30          25         130          25           25         30          25          30
+lifetime              30          25         100          25           25         30          25          30
```

Comment:
could you add a source for hydro lifetime to the end of core/input/generisdata_tech.prn?